src/components/LHNOptionsList/OptionRowLHNData.tsx

```diff
@@ -38,7 +38,10 @@ function OptionRowLHNData({
     const optionItemRef = useRef<OptionData>();
 
     const shouldDisplayViolations = ReportUtils.shouldDisplayViolationsRBRInLHN(fullReport, transactionViolations);
-    const shouldDisplayReportViolations = ReportUtils.isReportOwner(fullReport) && ReportUtils.hasReportViolations(reportID);
+    const isSettled = ReportUtils.isSettled(fullReport);
+    const shouldDisplayReportViolations = !isSettled && ReportUtils.isReportOwner(fullReport) && ReportUtils.hasReportViolations(reportID);
+    // We only want to show RBR for expense reports with transaction violations not for transaction threads reports.
+    const doesExpenseReportHasViolations = ReportUtils.isExpenseReport(fullReport) && !isSettled && ReportUtils.hasViolations(reportID, transactionViolations, true);
```

Comment:
@FitseTLT I didn't find this part in your proposal. Is this fixing a specific case? It seems to be causing a regression, RBR should be displayed only at the top level of reports i.e. only on the workspace chat.

Reply:
it says `expense reports and` and when you refer to expense reports in the context of RBR the only place we show red dots for reports is obviously in LHN, that's how I interpreted it but you can confirm with the original commenter 👍